src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpPreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -38,5 +39,19 @@ public sealed class CSharpPreferStreamAsyncMemoryOverloadsFixer : PreferStreamAs
                 });
             }
         }
+
+        protected override bool IsSystemNamespaceImported(IReadOnlyList<SyntaxNode> imports)
```

Comment:
@sharwell Does the code fixer actually need to check this? I though `AddImportAnnotation` will automatically no-op if it sees an existing using directive with same import.

Reply:
I had to add this check because otherwise, the import would be added at least 2 times. At least that's the behavior I saw in the unit tests, since the test source code gets analyzed more than once for each test.